go/cmd/yolo/main.go

```diff
@@ -25,19 +27,27 @@ import (
 	_ "github.com/jinzhu/gorm/dialects/sqlite"
 	circleci "github.com/jszwedko/go-circleci"
 	"github.com/peterbourgon/diskv"
-	ff "github.com/peterbourgon/ff/v2"
-	"github.com/peterbourgon/ff/v2/ffcli"
+	ff "github.com/peterbourgon/ff/v3"
 )
 
-var (
-	verbose        bool
-	logFormat      string
-	dbStorePath    string
-	withPreloading bool
-)
+type GlobalOptions struct {
+	verbose     bool
+	logFormat   string
+	dbStorePath string
+
+	server server
+}
+
+func (g *GlobalOptions) commonFlagsBuilder(fs *flag.FlagSet) {
+	fs.BoolVar(&g.verbose, "v", false, "increase log verbosity")
+	fs.StringVar(&g.logFormat, "log-format", "console", strings.Join(zapconfig.AvailablePresets, ", "))
+	fs.StringVar(&g.dbStorePath, "db-path", ":memory:", "DB Store path")
+}
+
+var optsGlobal = &GlobalOptions{}
 
 func main() {
-	err := yolo(os.Args)
+	err := yolo(os.Args[1:])
```

Comment:
Can you check how things are going when you run `yolo` without additional parameter?

Reply:
![image](https://user-images.githubusercontent.com/71719097/191003481-168e2c8c-ecd3-4b2b-92c9-12a1ea19a161.png)